working_agreement.md

```diff
@@ -28,14 +29,14 @@
 1. Building a diverse environment
     - Please be open to others perspectives, engage with them, and be willing to revise your own
     - We work to vocally support diverse perspectives
-    - To specifically amplify the work of Black, Indigenous, People-of-Color (BIPOC) researchers, we've committed to 
-      - Discussing an article written by a BIPOC author at least once a semester during lab meetings. Everyone is encouraged to lead a discussion
-      - Prioritize the citation of BIPOC authors in our writings
-      - Attend talks by BIPOC scientists at conferences and add the names of the speakers you enjoyed to [our spreadsheet](https://docs.google.com/spreadsheets/d/1wxUETDO0x9af4i4NA8BVdnoNyFvqVSpYhtlJa0azPrU/edit#gid=0) to use as inspiration for future seminar invitations and paper discussions
+    - To specifically amplify the work of researchers from underrepresented groups, we've committed to 
+      - Discussing an article about systemic barriers or racism at least twice a year during lab meetings. Everyone is encouraged to lead a discussion
+      - Prioritize the citation of underrepresented authors in our writings
+      - Attend talks by underrepresented scientists at conferences and share the names of the speakers you enjoyed to use as inspiration for future seminar invitations and paper discussions
        - Acknowledge our occupation of land that is the unceded territory of the Awaswas-speaking Uypi Tribe. 
-       - We will have biannual lab meetings to discuss our progress and future work in Diversity, Equity, Inclusion & Belonging, led by Malin
```

Comment:
We will have biannual lab meetings to discuss our progress and future work in Diversity, Equity, Inclusion, & Belonging, led by an outside facilitator with experience in DEIB